acceptance/testdata/secret/secret-repo.txtar

```diff
@@ -0,0 +1,76 @@
+# Setup environment variables used for testscript
+env REPO=$SCRIPT_NAME-$RANDOM_STRING
+
+# Use gh as a credential helper
+exec gh auth setup-git
+
+# Create a repository with a file so it has a default branch
+exec gh repo create $ORG/$REPO --add-readme --private
+
+# Defer repo cleanup
+defer gh repo delete --yes $ORG/$REPO
+
+# Clone the repo
+exec gh repo clone $ORG/$REPO
+cd $REPO
+
+# Create a repository secret
+exec gh secret set TESTSCRIPTS --body 'just a repository secret'
```

Comment:
I like that these don't do the variable business because they are already scoped. Nice.